packages/axelar-soroban-std/src/types.rs

```diff
@@ -1,8 +1,18 @@
-use soroban_sdk::{contracttype, Address};
+use soroban_sdk::{contracttype, Address, Env, String};
 
 #[contracttype]
 #[derive(Clone, Debug, Eq, PartialEq)]
 pub struct Token {
     pub address: Address, // TODO: check if this can be changed to a TokenClient type instead which is richer than Address, or a generic type implementing TokenInterface
     pub amount: i128,
 }
+
+const ZERO_ADDRESS: &str = "GAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAWHF";
+
+/// Returns Stellar's "dead" address, represented by the constant `ZERO_ADDRESS`.
+///
+/// # Reference
+/// - Stellar GitHub: https://github.com/stellar/js-stellar-base/blob/master/test/unit/address_test.js
+pub fn zero_address(env: &Env) -> Address {
```

Comment:
see below on how to extend types to add new methods
https://github.com/axelarnetwork/axelar-cgp-soroban/blob/main/packages/axelar-soroban-std/src/traits.rs#L31-L40

This allows creating a method like `Address::zero(env)` which is cleaner

you can put this in a new `address.rs` in the std

Reply:
good suggestion, updated